src/services/blog/request.js

```diff
@@ -0,0 +1,25 @@
+import { request } from 'undici';
+
+async function getTheDevRead(query) {
+  const data = [];
+
+  const tulisan = await requestDataByMedia('tulisan', query);
+  const media = await requestDataByMedia('web', query);
+  data.push(...tulisan);
+  data.push(...media);
+
+  return data;
+}
```

Comment:
ngga langsung gini aja?

```suggestion
async function getTheDevRead(query) {
  const tulisan = await requestDataByMedia("tulisan", query);
  const media = await requestDataByMedia("web", query);

  return [...tulisan, ...media];
}
```